elixir/apps/api/test/api/controllers/example_controller_test.exs

```diff
@@ -4,7 +4,7 @@ defmodule API.ExampleControllerTest do
   describe "echo/2" do
     test "returns error when not authorized", %{conn: conn} do
       conn = post(conn, "/v1/echo", %{"message" => "Hello, world!"})
-      assert json_response(conn, 401) == %{"error" => "invalid_access_token"}
+      assert json_response(conn, 401) == %{"error" => %{"reason" => "Unauthorized"}}
```

Comment:
This controller should be deleted, I left it there just as example that you can use when building the API

Reply:
👍 Wasn't sure whether you wanted that left in or not, but I'll get it removed.